lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -212,6 +212,16 @@ in the response if any changes to the database were made.
 
 =cut
 
+sub _ensure_trailing_newline {
+    my $template = shift;
+
+    # Automatically add a trailing line-break
+    if (length $template and $template !~ m/\n\z/) {
+        $template .= "\n";
+    }
+    return $template;
+}
```

Comment:
how about

```perl
sub _ensure_trailing_newline { "$_[0]\n" unless ($_[0] // '') =~ m/\n\z/ }
```

and then delete the `… // ''` from [line 240](https://github.com/os-autoinst/openQA/pull/2844/files#diff-db98705e995adce2fe11aac8d4c32eacR240)